torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -446,10 +481,14 @@ std::unique_ptr<caching::VectorizedTensorInfo> getVectorizedTensorValidationInfo
         vector_word_size.has_value(),
         "Non constant vector dimension found in ",
         out_tv);
-    tv_to_vector_word_size[out_tv] = vector_word_size.value();
-    tv_to_vector_word_size[in_tv] = vector_word_size.value();
+
+    auto tv_to_verify = out_tv->isFusionOutput() ? out_tv : in_tv;
```

Comment:
Ah, I should have added a comment here. The expression is just a Set op, so the extents of the input and output must be equal, so just checking either of them should be sufficient. 

The reason to check the output when it's a fusion output is just because the size information of fusion outputs (and inputs) are readily available without relying on the expression evaluator, so it's just for efficiency.

Reply:
Added a comment